test/browser/run-test.sh

```diff
@@ -28,7 +28,7 @@ fi
 
 export TEST_OS="${ID}-${VERSION_ID/./-}"
 # HACK: upstream does not yet know about rawhide
-if [ "$TEST_OS" = "fedora-37" ]; then
+if [ "$TEST_OS" = "fedora-38" ]; then
```

Comment:
38→36?

Do we want to add a fedora-37 image to the bots?

Reply:
38 -> 36 is what we always do for rawhide as we use `fedora-36` naughties for rawhide which might be something we want to change as it is a bit confusing.

Thinking about it, maybe we want to set TEST_OS=fedora-rawhide and set the exclusions / naughties using that as `rawhide` is a moving target.

I am not sure if we ever tested the Fedora Alpha? or Beta?